tests/Feature/Recorders/ServersTest.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+use Illuminate\Support\Facades\Config;
+use Illuminate\Support\Facades\Date;
+use Illuminate\Support\Facades\DB;
+use Laravel\Pulse\Events\SharedBeat;
+use Laravel\Pulse\Facades\Pulse;
+use Laravel\Pulse\Recorders\Servers;
+
+it('records server information', function () {
+    Config::set('pulse.recorders.'.Servers::class.'.server_name', 'Foo');
+    Date::setTestNow(Date::now()->startOfMinute());
+    event(app(SharedBeat::class));
+    Pulse::store();
+
+    expect(Pulse::ignore(fn () => DB::table('pulse_entries')->count()))->toBe(0);
+
+    $value = Pulse::ignore(fn () => DB::table('pulse_values')->sole());
+    expect($value->type)->toBe('system');
+    expect($value->key)->toBe('foo');
+    expect($value->timestamp)->toBe(Date::now()->startOfMinute()->timestamp);
+    $payload = json_decode($value->value);
+    expect($payload->name)->toBe('Foo');
+    expect($payload->cpu)->toBeGreaterThanOrEqual(0);
```

Comment:
Ideally, this would be `toBeGreaterThan`, but sometimes the CPU usage can be 0, which makes it impossible to tell whether it's a `null` that was cast to `0` or a real `0`.

I've tested it locally though :+1: